python/pyspark/sql/functions.py

```diff
@@ -153,7 +173,7 @@ def _():
 # math functions that take two arguments as input
 _binary_mathfunctions = {
     'atan2': 'Returns the angle theta from the conversion of rectangular coordinates (x, y) to' +
-             'polar coordinates (r, theta).',
+             'polar coordinates (r, theta). Units in radians.',
```

Comment:
I am unclear we should note this for every instance and users really gets confused as I see these use Scala/Java's built-in library. I wonder if there is an example that supports this differently in other libraries?